Client/Frontend/Browser/FaviconHandler.swift

```diff
@@ -0,0 +1,85 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+import Storage
+import SDWebImage
+import Deferred
+
+class FaviconHandler {
+    static let MaximumFaviconSize = 1 * 1024 * 1024 // 1 MiB file size limit
+
+    private var tabObservers: TabObservers!
+    private let backgroundQueue = OperationQueue()
```

Comment:
Should this `backgroundQueue` be just for this handler, or should we use it as a common one for other handlers too?

My intuition is we should share them — i.e. make them live with in `TabEventHandler.swift` or `TabEventHandlers.swift`. I don't know, what do you think?